conda_build/convert.py

```diff
@@ -763,7 +764,7 @@ def conda_convert(file_path, output_dir=".", show_imports=False, platforms=None,
     source_platform_architecture = '{}-{}' .format(source_platform, architecture)
 
     if 'all' in platforms:
-        platforms = ['osx-64', 'linux-32', 'linux-64', 'linux-ppc64',
+        platforms = ['osx-64', 'linux-32', 'linux-64', 'linux-ppc64','linux-s390x',
```

Comment:
Minor, but might be nice to swap the order of `ppc64le` and `s390x` here to keep the two `ppc*` arch's together in the list.

Reply:
@hartb : I have addressed this comment in the latest commit. Thanks!